docs/concepts/artifact.mdx

```diff
@@ -0,0 +1,239 @@
+---
+title: OCI artifact
+sidebar_position: 1
+---
+
+# Understanding OCI artifacts
+
+The Open Container Initiative (OCI) organization has played
+a crucial role in defining the industry-standard specifications for container
+formats and runtime and artifacts.
+OCI artifacts encompass an extensive variety of content types, from signatures,
+Software Bill of Materials (SBOMs), Helm charts, container images, to Open Policy Agent (OPA)
+bundles and more, thereby lending incredible flexibility to how containerized
+applications are packaged, delivered, and updated.
+
+This article will delve into OCI artifacts created using a manifest defined as a part of
+the OCI image specification.
+
+## The Anatomy of an OCI artifact
+
+The structure of an OCI Artifact includes an Image Manifest or an Image Index
```

Comment:
Suggest removing the index reference, per [this comment](https://github.com/oras-project/oras-www/pull/238#discussion_r1286499165)

> The persistence of an OCI Artifact is supported through the Image Manifest, which encompasses a collection of files, annotations, and an optional dependency reference to another artifact through the `subject` property.

Reply:
Index is a valid type and example has been removed. Article can be updated once we have better example.